tensorflow/core/platform/cloud/gcs_dns_cache.cc

```diff
@@ -26,6 +31,23 @@ namespace {
 constexpr char kStorageHost[] = "storage.googleapis.com";
 constexpr char kWwwHost[] = "www.googleapis.com";
 
+#ifndef _WIN32
```

Comment:
Nit: can you move the `#ifndef` inside the function to avoid duplicating the function prototype?

Reply:
Done.